lib/cache.module.ts

```diff
@@ -1,5 +1,5 @@
 import { DynamicModule, Module } from '@nestjs/common';
-import type { Cache as CoreCache } from 'cache-manager'
+import { createCache } from 'cache-manager';
```

Comment:
`createCache` is only used as a type, so the import should reflect that.

Maybe `@typescript-eslint/consistent-type-imports` should be enabled.

Reply:
Sorry, I didn't figure out how to use it, sometimes eslint is too painful for amateurs like me